op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -73,16 +74,20 @@ var taggedReleases = map[string]TaggedRelease{
 		ArtifactsHash: common.HexToHash("32e11c96e07b83619f419595facb273368dccfe2439287549e7b436c9b522204"),
 		ContentHash:   common.HexToHash("1cec51ed629c0394b8fb17ff2c6fa45c406c30f94ebbd37d4c90ede6c29ad608"),
 	},
+	ContractsV300Tag: {
+		ArtifactsHash: common.HexToHash("497e55cc6d9dfb74615d1bac0f4d05ae5ef995972689adcc036ee3adf4522677"),
+		ContentHash:   common.HexToHash("e2527ea1ddcd47bc60310baea4c3183de1520880561b6c7837213290b925a2e8"),
+	},
 }
 
 var _ embed.FS
 
 func IsSupportedL1Version(tag string) bool {
-	return tag == ContractsV200Tag
+	return tag == ContractsV300Tag
 }
 
 func IsSupportedL2Version(tag string) bool {
-	return tag == ContractsV170Beta1L2Tag
+	return tag == ContractsV300Tag
```

Comment:
Just to confirm - are we still able to run validations for V200 with these changes?

Reply:
yep!